index.js

```diff
@@ -135,7 +135,7 @@ function FsFuse(fs)
     .then(function(data)
     {
       data.copy(buffer)
-      cb(null, data.length)
+      cb(data.length)
```

Comment:
Are you sure of that? Some lines up first argument is used for errors, like normal Node.js errbacks does... Have you tested it, or checked against API docs? Maybe is it a bug on `node-fuse`?

Reply:
Yes and yes :-) .  When you say "node-fuse", I take it you mean fuse-bindings which is the package fs-fuse relies on.  [This](https://github.com/mafintosh/fuse-bindings#opsreadpath-fd-buffer-length-position-cb) is the API for read in fuse-bindings and as you can see, you simply return the length.

The way I figured this out was that I was getting no data passed through my implementation of fs-fuse.  I spent a lot of time trying to track down why, and finally realized that the cb(null, data.length) in read was passing back a zero where the length parameter belonged.  Changed it as above and the problem was solved.

Reply:
> Yes and yes :-) . When you say `node-fuse`, I take it you mean `fuse-bindings` which is the package `fs-fuse` relies on. [This](https://github.com/mafintosh/fuse-bindings#opsreadpath-fd-buffer-length-position-cb) is the API for `read` in `fuse-bindings` and as you can see, you simply return the length.

This reminds me about [this issue](https://github.com/mafintosh/fuse-bindings/issues/10) where I was advocating about changing the API. Not sure what could be the best thing here...

Reply:
Added comment and merged, thank you :-)